### ISSUE TITLE
Missing line after deltas

### ISSUE THREAD

**Issue by joshuarli** [joshuarli/ydiff#1]
This shouldn't happen. It's definitely a regression since original ydiff that I hadn't noticed till now.

<img width="281" alt="image" src="https://user-images.githubusercontent.com/14209781/85957393-9f6c9c80-b97c-11ea-863f-0b275117dd3b.png">

**Comment by joshuarli** [comment]
Tracked it down to the fix being this patch applied on top of bad commit `e31aa1533a7bf509fd226f4d6c2224fe23b9d01c`:

```diff
diff --git a/ydiff b/ydiff
index af25ddc..12f43d7 100755
--- a/ydiff
+++ b/ydiff
@@ -357,6 +357,8 @@ class DiffMarker(object):
                     else:
                         left = _fit_with_marker_mix(left)
                         right = _fit_with_marker_mix(right)
+                else:
+                    right = f"{COLORS['reset']}{right}"

                 # Need to wrap long lines, so here we'll iterate,
                 # shaving off `width` chars from both left and right
```

**Comment by joshuarli** [comment]
Fixed in 7fe863a136e12b6957476179dabe6a0ed39aedfc.
